### PR TITLE
Install Cloudinary v2

### PR DESCRIPTION
**Pull request by dylancherry98**
Re-did the Cloudinary installation.

Please let me know what you think.

@nafriedman @MmeIsabelle @Tien9320

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -35,11 +35,13 @@ gem 'autoprefixer-rails', '10.2.5'
 gem 'font-awesome-sass'
 gem 'simple_form'
 gem 'faker'
+gem 'cloudinary', '~> 1.16.0'
 
 group :development, :test do
   gem 'pry-byebug'
   gem 'pry-rails'
-  gem 'dotenv-rails'
+  gem 'dotenv-rails', groups: [:development, :test]
```
Comment (nafriedman) [inline Gemfile:43]: Put this outside this group and right above the Cloudinary gem

Comment (nafriedman) [reply Gemfile:43]: Actually just put it outside this group

Comment (dylancherry98) [reply Gemfile:43]: Done!

**Comment by dylancherry98** [comment]
Don't push yet! Getting an error in my web browser now:
"An error has occurred, this and all later migrations canceled: PG::DuplicateTable: ERROR: relation "active_storage_blobs" already exists"

**Comment by dylancherry98** [comment]
Errors should be fixed! Please review.

**Pull request merged**